scss/_variables.scss

```diff
@@ -169,11 +169,9 @@ $spacers: (
   1: $spacer * .25,
   2: $spacer * .5,
   3: $spacer,
-  4: $spacer * 1.25,
-  5: $spacer * 2,
-  6: $spacer * 3,
-  7: $spacer * 6,
-  8: $spacer * 9,
+  4: $spacer * 1.5,
```

Comment:
why not use fixed step, for example `.25`: `1: $spacer * .5, 2: $spacer * .75, 3: $spacer * 1, 4: $spacer * 1.25, 5: $spacer * 1.5, 6: $spacer * 1.75,` ?